src/test/java/co/cask/gcp/datastore/source/EntityToRecordTransformerTest.java

```diff
@@ -194,8 +201,7 @@ public void testTransformDifferentType() {
       .set("field", "field_value")
       .build();
 
-    thrown.expect(ClassCastException.class);
-    thrown.expectMessage("java.lang.String cannot be cast to java.lang.Long");
+    thrown.expect(IllegalStateException.class);
```

Comment:
it looks strange for me to see IllegalStateException from transformers. It should be IllegalArgumentException or cdap UnexpectedFormatException everywhere to make it possible to handle properly. I understand it is legacy, just saying.

Reply:
Agree, changed for this case only since this one is newly added exception.